notebook/templates/tree.html

```diff
@@ -7,6 +7,10 @@
   {% block favicon %}
   <link rel="icon" type="image/x-icon" href="{{ base_url | escape }}static/favicons/favicon.ico" class="favicon">
   {% endblock %}
+
+  {% if custom_css %}
+  <link rel="stylesheet" type="text/css" href="{{ base_url | escape }}custom/custom.css">
+  {% endif %}
```

Comment:
Should this also be added to the `consoles.html` page?

https://github.com/jupyter/notebook/blob/main/notebook/templates/consoles.html